### PR TITLE
ElevationProfile_optimize performance 

### PR DESCRIPTION
**Pull request by norbertK**

### CODE REVIEW

**Comment by taulinger** [comment]
Can confirm, works much faster now 🥳

File: src/modules/elevationProfile/components/panel/ElevationProfile.js [hunk 1]
```diff
@@ -331,10 +332,32 @@ export class ElevationProfile extends MvuElement {
...
+							currentColors[selectedAttribute] = {};
+						}
+						if (!currentColors[selectedAttribute].backgroundColor) {
+							if (context.chart.chartArea) {
+								currentColors[selectedAttribute].backgroundColor = this._getBackground(context.chart, elevationData, selectedAttribute);
+							} else {
+								return ElevationProfile.BACKGROUND_COLOR;
+							}
+						}
+						return currentColors[selectedAttribute].backgroundColor;
```
Comment (thiloSchlemmer) [inline src/modules/elevationProfile/components/panel/ElevationProfile.js:346]: Do we need these checks? Can we reduce this to `ElevationProfile.BACKGROUND_COLOR`

**Review by thiloSchlemmer — COMMENTED** [review]
see inline comments

File: src/modules/elevationProfile/components/panel/ElevationProfile.js (hunk 1 above)
Comment (norbertK) [reply src/modules/elevationProfile/components/panel/ElevationProfile.js:346]: hm - the background part is entered twice for every point (about 6000 times over long distances), but it doesn't 'do' anything, it just returns what is computed in _getBackground (normally ElevationProfile.BACKGROUND_COLOR or _getTextTypeGradient for surface) once.

**Review by thiloSchlemmer — APPROVED** [review]
Thx

**Pull request merged**